### PR TITLE
FLEX-1612 ~ Feat protocol adapter iec61850

### PR DESCRIPTION
**Pull request by thijlboonen**
Adds feature setLight

### CODE REVIEW

File: osgp-core-db-api-iec61850/src/main/java/com/alliander/osgp/core/db/api/iec61850/entities/Device.java [hunk 1]
```diff
@@ -0,0 +1,161 @@
+/**
```
Comment (thijlboonen) [inline osgp-core-db-api-iec61850/src/main/java/com/alliander/osgp/core/db/api/iec61850/entities/Device.java]: Copied and dressed down version of the platform device, no new code here

File: osgp-core-db-api-iec61850/src/main/java/com/alliander/osgp/core/db/api/iec61850/entities/DeviceOutputSetting.java [hunk 2]
```diff
@@ -0,0 +1,80 @@
+/**
```
Comment (thijlboonen) [inline osgp-core-db-api-iec61850/src/main/java/com/alliander/osgp/core/db/api/iec61850/entities/DeviceOutputSetting.java]: Copied code

File: osgp-core-db-api-iec61850/src/main/java/com/alliander/osgp/core/db/api/iec61850valueobjects/RelayFunction.java [hunk 3]
```diff
@@ -0,0 +1,16 @@
+/**
```
Comment (thijlboonen) [inline osgp-core-db-api-iec61850/src/main/java/com/alliander/osgp/core/db/api/iec61850valueobjects/RelayFunction.java]: Copied code

File: osgp-core-db-api-iec61850/src/main/java/com/alliander/osgp/core/db/api/iec61850valueobjects/RelayType.java [hunk 4]
```diff
@@ -0,0 +1,33 @@
+/**
```
Comment (thijlboonen) [inline osgp-core-db-api-iec61850/src/main/java/com/alliander/osgp/core/db/api/iec61850valueobjects/RelayType.java]: Copied code

File: osgp-core-db-api/src/main/java/com/alliander/osgp/core/db/api/application/services/DeviceDataService.java [hunk 5]
```diff
@@ -1,40 +0,0 @@
-/**
```
Comment (thijlboonen) [inline osgp-core-db-api/src/main/java/com/alliander/osgp/core/db/api/application/services/DeviceDataService.java]: Unused Class

File: osgp-core-db-api/src/main/java/com/alliander/osgp/core/db/api/entities/Device.java [hunk 6]
```diff
@@ -1,74 +0,0 @@
-/**
- * Copyright 2015 Smart Society Services B.V.
```
Comment (thijlboonen) [inline osgp-core-db-api/src/main/java/com/alliander/osgp/core/db/api/entities/Device.java]: Unused Class

File: osgp-core-db-api/src/main/java/com/alliander/osgp/core/db/api/entities/Organisation.java [hunk 7]
```diff
@@ -1,42 +0,0 @@
-/**
```
Comment (thijlboonen) [inline osgp-core-db-api/src/main/java/com/alliander/osgp/core/db/api/entities/Organisation.java]: Unused Class

File: osgp-core-db-api-iec61850/src/main/java/com/alliander/osgp/core/db/api/iec61850/entities/Ssld.java [hunk 8]
```diff
@@ -0,0 +1,144 @@
+/**
```
Comment (thijlboonen) [inline osgp-core-db-api-iec61850/src/main/java/com/alliander/osgp/core/db/api/iec61850/entities/Ssld.java]: Copied class from device. Only findByRelayType() and getDeviceOutputSettingForIndex() are new here

File: osgp-protocol-adapter-iec61850/src/main/java/com/alliander/osgp/adapter/protocol/iec61850/device/responses/EmptyDeviceResponse.java [hunk 9]
```diff
@@ -15,11 +15,20 @@
     private DeviceMessageStatus status;
 
     public EmptyDeviceResponse(final String organisationIdentification, final String deviceIdentification,
+            final String correlationUid) {
+        super(organisationIdentification, deviceIdentification, correlationUid);
+    }
+
+    public EmptyDeviceResponse(final String organisationIdentification, final String deviceIdentification,
```
Comment (kevinsmeets) [inline osgp-protocol-adapter-iec61850/src/main/java/com/alliander/osgp/adapter/protocol/iec61850/device/responses/EmptyDeviceResponse.java]: Please use a dedicated function to handle the GetSatus-response. The EmptyDeviceResponse() function should be used for device responses that contain only 'OK'. Functions that return values, like GetStatus, GetConfiguration, etc. should implement a dedicated function to handle the response.

**Pull request merged**